### ISSUE TITLE
flickr.js should have umbraBehaviorFinished function

### ISSUE THREAD

**Issue by ldko** [internetarchive/umbra#47]
Processing URIs with flickr.js results in logged messages like:
`2015-11-19 14:27:25,481 15683 ERROR WebsockThread9200-VdPjA5 umbra.behaviors.Behavior.notify_of_result(behaviors.py:89) chrome message doesn't look like a boolean result! {'id': 5222, 'result': {'exceptionDetails': {'text': 'Uncaught ReferenceError: umbraBehaviorFinished is not defined', 'stackTrace': [{'functionName': '', 'scriptId': '1351', 'lineNumber': 1, 'columnNumber': 1, 'url': ''}, {'functionName': 'InjectedScript._evaluateOn', 'scriptId': '176', 'lineNumber': 904, 'columnNumber': 140, 'url': ''}, {'functionName': 'InjectedScript._evaluateAndWrap', 'scriptId': '176', 'lineNumber': 837, 'columnNumber': 34, 'url': ''}, {'functionName': 'InjectedScript.evaluate', 'scriptId': '176', 'lineNumber': 693, 'columnNumber': 21, 'url': ''}], 'column': 0, 'line': 1, 'url': ''}, 'wasThrown': True, 'result': {'subtype': 'error', 'type': 'object', 'description': 'ReferenceError: umbraBehaviorFinished is not defined\n    at <anonymous>:1:1\n    at Object.InjectedScript._evaluateOn (<anonymous>:904:140)\n    at Object.InjectedScript._evaluateAndWrap (<anonymous>:837:34)\n    at Object.InjectedScript.evaluate (<anonymous>:693:21)', 'objectId': '{"injectedScriptId":2,"id":1157}', 'className': 'ReferenceError'}}}`

**Comment by nlevitt** [comment]
Yes, thanks! We'll get to it at some point I'm sure, but if you want to accelerate the process, feel free to supply a pull request.

**Comment by mitchell-frost** [comment]
I am new to open source. Can I work on this?
